src/hb-aat-layout.cc

```diff
@@ -301,3 +309,29 @@ _hb_aat_language_get (hb_face_t *face,
 {
   return _get_ltag (face).get_language (i);
 }
+
+/**
+ * hb_aat_get_feature_settings:
+ * @face:            a font face.
+ * @identifier:      aat feature id you are querying.
```

Comment:
```suggestion
 * @identifier:      AAT feature id you are querying, for example 1 for "Ligatures" feature, 37 for the "Lower Case" feature, 38 for the "Upper Case" feature, etc.
```

Reply:
Applied. Thanks

Reply:
I would be in favour of having an enum, or at least clearly specifying a source for where these values should come from / what the reference would be for them.

Reply:
Turned into enum.